.travis/script.sh

```diff
@@ -59,6 +59,7 @@ python --version || die
         -DOPENTHREAD_CONFIG_DIAG_ENABLE=1                 \
         -DOPENTHREAD_CONFIG_DNS_CLIENT_ENABLE=1           \
         -DOPENTHREAD_CONFIG_ECDSA_ENABLE=1                \
+        -DOPENTHREAD_CONFIG_IP6_ENABLE_FRAGMENTATION=1    \
```

Comment:
nit: can we rename this to `OPENTHREAD_CONFIG_IP6_FRAGMENTATION_ENABLE` for consistency?

Reply:
naturally. changed.